gender_novels/corpus_gen.py

```diff
@@ -47,22 +51,34 @@ def generate_corpus_gutenberg():
         cache.populate()
         print("Done populating cache")
     # go through all books in Keith's thing
-    bookshelf = str(Path(current_dir, INITIAL_BOOK_STORE, r"*.txt"))
-    print("Searching folder", bookshelf)
-    books = glob.iglob(bookshelf)
+
     number_books = 0
-    print('')
     start_time = time.time()
-    for book in books:
+
+    # Generate filepaths
+    # gutenberg_id.txt means ascii file (which works fine for our purposes)
+    # gutenberg_id-0.txt means utf8 file (preferred)
+    filepaths = []
+    for gutenberg_id in range(200):
+        novel_directory = generate_gutenberg_rsync_path(gutenberg_id)
+        utf8_path = novel_directory.joinpath(f'{gutenberg_id}-0.txt')
+        if os.path.isfile(utf8_path):
+            filepaths.append(utf8_path)
+            continue
+        ascii_path = novel_directory.joinpath(f'{gutenberg_id}.txt')
+        if os.path.isfile(ascii_path):
+            filepaths.append(ascii_path)
```

Comment:
Maybe also consider what to do if there is no book with a certain Gutenberg ID, although it seems that every ID corresponds to a book.